processors/auditd/sessiontracker/README.md

```diff
@@ -0,0 +1,126 @@
+# Session Tracking Using Audito Maldito.
```

Comment:
Normally I find these Codacy warnings to be useless - but in this case they are generally correct.

- Section headings do not require punctuation (in other words, you do not need to end them with a period)
- Section headings and code fences (the three backtick characters) should have an empty newline between them. For example:

```
## Some stuff

Hello world, this is the some stuff section.

'''
This is a code fence (I used single quotes here
so I do not need to escape them)
'''

## Another section

Here is another section.
```

Note how the section headings and the code fences are separated by empty newlines.